FlowCrypt/Controllers/Inbox/InboxRenderable.swift

```diff
@@ -65,4 +63,37 @@ extension InboxRenderable {
         self.date = date ?? Date()
         self.wrappedType = .thread(thread)
     }
+
+    @MainActor func hideErrorLabel() {
+
+    }
+
+    private static func messageTitle(with thread: MessageThread, and folderPath: String) -> String {
+        let sentPath = "SENT"
+        guard let myEmail = DataService.shared.email else { return "" }
+
+        // for now its not exactly clear how titles on other folders should looks like
+        // so in scope of this PR we are applying this title presentation only for "sent" folder
+        if folderPath == sentPath {
```

Comment:
I think we can use `MessageLabelType.sent.value` here instead of just string `SENT`, as we already have all message labels/folders stored in MessageLabelType.